src/DateRangePicker.elm

```diff
@@ -97,9 +99,11 @@ import Time.Extra as TE
 
 -}
 type alias Config =
-    { allowFuture : Bool
+    { actionButtons : Calendar.ActionButtons
+    , allowFuture : Bool
     , applyRangeImmediately : Bool
     , class : String
+    , hintText : Calendar.HintText
```

Comment:
I'd strongly suggest we use a single data structure to hold translations, for both button captions and hint texts.

Reply:
Ok, it is in on record now.